reactioncommerce/reaction
# Fix "Set Custom Value" on Avalara Usage Type

Resolves #2101

How to test:
- Login as an admin
- With Avalara enabled, Select "Manage" for a user
- Choose "SET CUSTOM VALUE" under Tax Settings
- A "Custom Value" input should appear. 
- Type in a code and save.
- Ensure saving works as it should
